integration/component/05_install_dev_r10k.rb

```diff
@@ -0,0 +1,37 @@
+test_name "Install PE r10k" do
+
+  step "Install PE r10k" do
+    product = 'pe-r10k'
+
+    opts = {
+      :puppet_collection       => 'PC1',
+      :pe_r10k_sha     => ENV['SHA'],
+      :pe_r10k_version => ENV['SUITE_VERSION'] || ENV['SHA']
+    }
+
+    client = find_only_one('client')
+
+    install_pe_client_tools_on(client, opts)
+
+  end
+
+end
+
+# taken from beaker-pe/lib/beaker-pe/pe-client-tools/install_helper.rb
+def install_pe_client_tools_on(hosts, opts = {})
```

Comment:
Could we use `install_pe_r10k_on`? Or even, `install_pe_product_on` and pass in product as an arg?